isis/src/base/objs/SpiceRotation/SpiceRotation.cpp

```diff
@@ -2296,7 +2312,7 @@ namespace Isis {
     ktotal_c("ck", (SpiceInt *) &count);
 
     // Downsize the loaded cache
-    if ((p_source == Memcache) && p_minimizeCache == Yes) {
+    if ((p_source == Memcache) && p_minimizeCache == Yes) { // FIXME me (skip this if: minimizeCache == no)
```

Comment:
Remove all FIXME comments. They don't need to be fixed, actually, I just use 'FIXME' as a flag that I need to take something out before comitting. (oops)